scripts/make_metacal_catalog.py

```diff
@@ -0,0 +1,188 @@
+"""
+make_metacal_catalog.py
+
+Save catalogs to parquet from metacal runs across available patches.
+"""
+import os
+import re
+
+import numpy as np
+from astropy.table import hstack
+
+from lsst.daf.persistence import Butler
+from lsst.daf.persistence.butlerExceptions import NoResults
+
+
+def _ensure_butler_instance(butler_or_repo):
+    if not isinstance(butler_or_repo, Butler):
+        return Butler(butler_or_repo)
+    return butler_or_repo
+
+
+def generate_metacal_catalog(output_dir, butler, tract, patches=None,
+                             overwrite=True, verbose=False,
+                             filename_prefix='metacal',
+                             parquet_engine='pyarrow',
+                             **kwargs):
+    """Save catalogs to parquet from individual metacal patches.
```

Comment:
"""Save catalogs to parquet from individual metacal tract.

Reply:
hum, this scripts outputs a parquet file for each (tract,patch),  they are named "metacal_1450_11.parquet" or something like that. It's the same logic as in make_object_catalog.py
Is that ok? or would you rather output a single file per tract?